### ISSUE TITLE
Voronoi treemap

### ISSUE THREAD

**Issue by lazarusA** [DanielVandH/DelaunayTriangulation.jl#87]
any plans for this? 

https://github.com/Kcnarf/d3-voronoi-treemap

**Comment by DanielVandH** [comment]
Never heard of a Voronoi treemap! While I don't have any plans for it currently, I do have plans for weighted Delaunay triangulations and weighted Voronoi diagrams, which are the key components of the algorithm. Maybe in a couple of months I can revisit this again? The next features I'm prioritising, though, are spatial sorting and curved boundaries, so might be a bit longer unless there's a lot more interest for it.

**Comment by lazarusA** [comment]
Indeed, they not pretty common in the science domain, but rather more in the political one. Recently, there is an increase in his use to convey messages to the general public. 

Good to know that some of the things necessary to make this work are already on the roadmap. Thanks.
